source/examples/generated/flutter/read_write_data_test.snippet.filter-iterable.dart

```diff
@@ -0,0 +1,3 @@
+final listOfNames = ['Luke', 'Leia'];
+final matchingRealmObjects =
+    realm.query<Person>(r'name IN $0', [listOfNames]);
```

Comment:
If this is a typo and you just want to batch these suggestions, offering this here for convenience so you don't have to regenerate w/Bluehawk.
```suggestion
    realm.query<Person>('name IN $0', [listOfNames]);
```